convert2rhel/checks.py

```diff
@@ -95,56 +97,59 @@ def check_convert2rhel_latest():
     repo_dir = tempfile.mkdtemp(prefix="convert2rhel_repo.", dir=utils.TMP_DIR)
     utils.mkdir_p(repo_dir)
     repo_path = os.path.join(repo_dir, "convert2rhel.repo")
-    store_content_to_file(filename=repo_path, content=repo_content)
+    store_content_to_file(filename=repo_path, content=REPO_CONTENT)
 
     cmd.append("--setopt=reposdir=%s" % repo_dir)
     cmd.append("convert2rhel")
 
     raw_output_convert2rhel_versions, return_code = run_subprocess(cmd, print_output=False)
 
-    if return_code != 0:
-        logger.warning(
-            "Couldn't check if this is the latest version of convert2rhel\n"
-            "repoquery failed %s, %s" % (return_code, raw_output_convert2rhel_versions)
-        )
-        return
-
     PKG_NEVR = r"\b(\S+)-(?:([0-9]+):)?(\S+)-(\S+)\b"
     convert2rhel_versions = re.findall(PKG_NEVR, raw_output_convert2rhel_versions, re.MULTILINE)
-    latest_version = ("0", "0.00", "0")
+    latest_available_version = ("0", "0.00", "0")
 
+    # This loop will determin the latest_available_version in the convert2rhel yum repo
+    # it assigns the epoch ,version, and release ex:("0" "0.26" "1.el7") to the latest_available_version var.
     for package_version in convert2rhel_versions:
-        # rpm.lableCompare(pkg1, pkg2) compare two kernel version strings and return
-        # -1 if str2 is greater then str1, 0 if they are equal, 1 if str1 is greater the str2
-        ver_compare = rpm.labelCompare(package_version[1:], latest_version)
+        # rpm.lableCompare(pkg1, pkg2) compare two package version strings and return
+        # -1 if lastest_version is greater then package_version, 0 if they are equal, 1 if package_version is greater the lastest_version
```

Comment:
```suggestion
        # -1 if lastest_version is greater than package_version, 0 if they are equal, 1 if package_version is greater than lastest_version
```

Reply:
I will definitely get this fixed thank you for spotting this!